aiida_common_workflows/workflows/relax/gpaw/protocol.yml

```diff
@@ -0,0 +1,41 @@
+oxides_validation:
+  name: 'oxides_validation'
+  description: 'Protocol to run oxides validation study.'
+  parameters:
+    calculator:
+      name: 'gpaw'
+      args:
+        xc: 'PBE'
+        mode:
+          '@function': 'PW'
+          args:
+            ecut: 800
+        occupations:
+          name: 'fermi-dirac'
+          width: 0.06122
+    optimizer:
+      name: 'BFGS'
+    kpoints_options:
+      gamma: True
+  kpoint_distance: 0.06
+
+moderate:
+  name: 'moderate'
```

Comment:
Could you please add a `fast` and `precise` protocol as well?

Reply:
I'm not sure if I could add something without more testing or at least a few checks of some sort. Are these requirements for `protocol.yml`?

Reply:
I don't remember if this is a "hard" requirement, but that was the setup of the original implementations and paper. Each implementation currently provides a `fast` and `precise` protocol. Note that many of these are not actually tested in great detail and the values were actually chosen just based on experience of the developer to what seemed reasonable. If you also have an idea of what would be reasonable values, please add them. It doesn't need to be perfect and we can always improve it in the future. If you really don't have a clue, it is ok to leave them out for now.